### ISSUE TITLE
Squawk CLI fails to parse SQL migrations without explicit index names

### ISSUE THREAD

**Issue by natalie-zamani** [sbdchd/squawk#167]
Given a SQL migration file with a create index statement, Squawk fails to parse the file if it lacks an explicit index name with the following error, even though Postgres allows omitting explicit index naming:

Sample SQL:
```sql
CREATE INDEX ON FOO(BAR);
```

Error:
```bash
Problem linting SQL files: Squawk schema failed to parse Postgres response. This indicates a bug with Squawk. Please report this error to https://github.com/sbdchd/squawk. Schema error: missing field `idxname` at line 1 column 439
```

**Comment by sbdchd** [comment]
Thanks for the report!

Should be fixed in 0.8.1
